test/integration/garbagecollector/garbage_collector_test.go

```diff
@@ -461,6 +461,9 @@ func setupRCsPods(t *testing.T, gc *garbagecollector.GarbageCollector, clientSet
 	}
 	orphan := false
 	switch {
+	case options.OrphanDependents == nil && options.PropagationPolicy == nil && len(initialFinalizers) == 0:
```

Comment:
what about just moving this to the end of the switch?

Reply:
I was trying to keep it in the same location to minimize the logical diff from before